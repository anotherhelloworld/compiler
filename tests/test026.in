var
  set1: set of 1..10;
  set2: set of 'a'..'z';
  set3: set of pear..orange;