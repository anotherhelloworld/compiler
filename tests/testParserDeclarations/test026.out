Procedure   foo   0
   begin
   end

